trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/premium-payments-errors/initiate-record-correction-request-error.ts

```diff
@@ -0,0 +1,47 @@
+import { PREMIUM_PAYMENTS_TABLE_ERROR_HREF } from '../../../../constants/premium-payments-table-error-href';
+import { INITIATE_RECORD_CORRECTION_ERROR_KEY } from '../../../../constants/premium-payment-tab-error-keys';
+import { InitiateRecordCorrectionRequestErrorKey } from '../../../../types/premium-payments-tab-error-keys';
+import { ErrorSummaryViewModel } from '../../../../types/view-models';
+
+const { MULTIPLE_FEE_RECORDS_SELECTED, NO_FEE_RECORDS_SELECTED, INVALID_STATUS } = INITIATE_RECORD_CORRECTION_ERROR_KEY;
+
+const initiateRecordCorrectionRequestErrorMap: Record<InitiateRecordCorrectionRequestErrorKey, ErrorSummaryViewModel> = {
+  [MULTIPLE_FEE_RECORDS_SELECTED]: {
+    text: 'Select one fee for a fee record correction request',
+    href: PREMIUM_PAYMENTS_TABLE_ERROR_HREF,
+  },
+  [NO_FEE_RECORDS_SELECTED]: {
+    text: 'Select a record to create a record correction request',
```

Comment:
I'm only seeing E2E coverage for this error message, can we add for the others?

Reply:
Added 👍